### PR TITLE
v2.9.1

### PR DESCRIPTION
**Pull request by ngam**
Closes #240 
Closes #245 
Closes #244 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,7 +24,7 @@ build:
   # Tensorflow still hasn't updated from C++14.
```
Comment (h-vetinari) [inline recipe/meta.yaml:24]: This comment seems out of date now? Last status was that the newer abseil builds are all C++17, right?

**Review by h-vetinari — APPROVED** [review]
I only superficially checked the absl assert patch (that the conditions are translated correctly, not what macros are being replaced), but basically LGTM. Thanks!

**Comment by xhochy** [comment]
@ngam Looks all good. I will start the OSX builds. I'm happy to do all of them but that will probably take until the end of the week to finish

File: recipe/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipe/meta.yaml:24]: I think your understanding is correct. Lets get this merged in then we can move it forward.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -26,7 +24,7 @@ build:
...
   # Newer versions of abseil_cpp require C++17
   # This may be a possibility in 2.9.0
```
Comment (hmaarrfk) [inline recipe/meta.yaml:26]:
```suggestion
  # We specify a version of abseil_cpp that is compatible with C++17
```

**Comment by hmaarrfk** [comment]
I can start these builds after pytorch.

**Comment by hmaarrfk** [comment]
You are correct, I can only build linux.

> xhochy decided to rebuild them anyway.

I think it would help if you build them. Ultimately, the manual upload process is really flawed, and not really a good one to follow. They probably felt it was more accountable if they built the packages themselves.

**Comment by hmaarrfk** [comment]
you can likely adapt the `README.md` to find the osx builds instead of the linux64

**Comment by hmaarrfk** [comment]
Usually there is something like:
```
[0 / 12] [Prepa] Creating source manifest for //tensorflow/lite/python:tflite_convert
[83 / 2,088] Compiling mlir/tools/mlir-tblgen/CodeGenHelpers.cpp; 7s local ... (3 actions running)
[99 / 2,106] Compiling mlir/lib/TableGen/Pattern.cpp; 11s local ... (3 actions running)
```
But i can't find them in your log.

**Comment by hmaarrfk** [comment]
i'm not sure, it might be a terminal thing. I'm just worried that things are not getting compiled accidentally.

**Comment by hmaarrfk** [comment]
What did you change?

**Comment by hmaarrfk** [comment]
I see. I typically do
```
| tee log.txt
```

or something similar. so i get both

**Comment by hmaarrfk** [comment]
the builds look good.!

**Comment by hmaarrfk** [comment]
There may be a few things that depend on tensorflow. Lets wait to have everything before merging. It seems like you are getting really close.

**Comment by hmaarrfk** [comment]
I guess people can wait. No issue. Enjoy your time away!

**Comment by hmaarrfk** [comment]
"i guess" is more me agreeing with you that people should just wait.

From a conda-forge standpoint, it is annoying to have packages that are half upgraded.

**Comment by hmaarrfk** [comment]
Linux logs
[log_files.zip](https://github.com/conda-forge/tensorflow-feedstock/files/9137398/log_files.zip)

Uploaded to https://anaconda.org/mark.harfouche under the label `cfep03`

**Comment by hmaarrfk** [comment]
Thank you @ngam, @h-vetinari
